src/api-service/__app__/agent_can_schedule/function.json

```diff
@@ -0,0 +1,22 @@
+{
+	"scriptFile": "__init__.py",
+	"bindings": [
+		{
+			"authLevel": "anonymous",
+			"type": "httpTrigger",
+			"direction": "in",
+			"name": "req",
+			"methods": [
```

Comment:
For now, I suggest just not routing the unused methods (I assume these are just here from copy/paste).